### DISCUSSION TITLE
Why is my laptop exactly at 90% charge when al dente is set to 80?

### DISCUSSION THREAD

**Discussion by KernelBypass** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1153] [General]
Started using it recently, and this morning found my laptop hovering at higher charge level than what the setting commands.
Is this a bug?

```
AlDente Version 1.24.1 (48)
MacOS 14.2.1
Laptop M3Max 40-core 128GB
```

**Comment by MatthiasKerbl** [comment]
Hi @KernelBypass,

Please answer the following questions so that we can assess this issue better:

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots or screen recordings to help explain your problem.

**MacBook (please complete the following information):**
 - MacBook model (https://support.apple.com/en-us/HT201608)
 - macOS version (https://support.apple.com/en-us/HT201260)
 - AlDente Pro or AlDente Free?
 - AlDente Version?
 - List of all enabled/disabled settings in AlDente:

**Additional context**
Add any other context about the problem here.
